Lombiq.Vsix.Orchard/Services/LogWatcher/IBlinkStickManager.cs

```diff
@@ -1,10 +1,27 @@
-﻿namespace Lombiq.Vsix.Orchard.Services.LogWatcher
+using System;
+
+namespace Lombiq.Vsix.Orchard.Services.LogWatcher
 {
-    public interface IBlinkStickManager
+    /// <summary>
+    /// An interface for blink stick led operations.
```

Comment:
LED a többi helyen is.

```suggestion
    /// An interface for BlinkStick LED operations.
```